DEVELOPER_GUIDE.md

```diff
@@ -74,6 +83,74 @@ This is typically created in PyCharm IDE by visiting the `Python Interpreter`, s
 `
 In order to run tests within the PyCharm IDE, ensure the `Python Integrated Tools` / `Testing` / `Default Test Runner` is set to `pytest`.
 
+## Running Workloads
+
+### Installation  
+
+Download the latest release of OpenSearch from https://opensearch.org/downloads.html. If you are using WSL, make sure to download it into your `/home/<user>` directory instead of `/mnt/c`. 
+```
+wget https://artifacts.opensearch.org/releases/bundle/opensearch/<x.x.x>/opensearch-<x.x.x>-linux-x64.tar.gz
+tar -xf opensearch-x.x.x-linux-x64.tar.gz
+cd opensearch-x.x.x
+```
+NOTE: Have Docker running in the background for the next steps. Refer to the installation instructions [here](https://docs.docker.com/compose/install/).
+
+### Setup 
```

Comment:
Would be more descriptive here to differentiate that this is for OpenSearch setup and not OSB